tgui/packages/tgui/styles/components/Tabs.scss

```
@@ -126,6 +126,39 @@ $fg-map: colors.$fg-map !default;
+    background: linear-gradient(
+      90deg,
+      rgba(251, 214, 8, 0.1),
+      rgba(251, 214, 8, 0.9),
+      rgba(251, 214, 8, 0.1)
+    );
+    background-size: 200% 100%;
+    animation: TabCustomShimmer 3s linear infinite;
+  }
+}
+
+@keyframes TabCustomShimmer {
```

Stylelint (17.14.0)

[error] Expected keyframe name "TabCustomShimmer" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)